### DISCUSSION TITLE
Third Party Library

### DISCUSSION THREAD

**Discussion by TakayasuNasu** [DaiSugi01/Queens-game#40] [Ideas]
Do you have a library that you would like to use?
The following is a collection of Library information.

https://github.com/vsouza/awesome-ios

Until now

1. [realm/realm\-cocoa: Realm is a mobile database: a replacement for Core Data & SQLite](https://github.com/realm/realm-cocoa)

I would like to use the following two libraries for testing.

1. [Quick/Quick: The Swift \(and Objective\-C\) testing framework\.](https://github.com/Quick/Quick)
1. [Quick/Nimble: A Matcher Framework for Swift and Objective\-C](https://github.com/Quick/Nimble)

**Comment by DaiSugi01** [comment]
I have nothing.

**Comment by cookie777** [comment]
For swiping collection view cell !
https://github.com/SwipeCellKit/SwipeCellKit
